### PR TITLE
[FIX] spreadsheet: add show values option for chart

### PR DESCRIPTION
**Pull request by anhe-odoo**
## Task Description

This task aims to add the possibility to show the value of the series on the chart (above the point for scatter/line chart, above the bar for bar/combo chart and inside part for pie chart).

## Related Task

- Task: [3953835](https://www.odoo.com/odoo/project/2328/tasks/3953835?cids=1)
- https://github.com/odoo/o-spreadsheet/pull/4324

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/spreadsheet/static/src/chart/odoo_chart/odoo_bar_chart.js [hunk 1]
```diff
@@ -102,5 +102,12 @@ function getBarConfiguration(chart, labels, locale) {
         config.options.scales.x.stacked = true;
         config.options.scales.y.stacked = true;
     }
+
+    config.options.plugins.chartShowValuesPlugin = {
+        showValues: chart.showValues,
```
Comment (rrahir) [inline addons/spreadsheet/static/src/chart/odoo_chart/odoo_bar_chart.js:107]: missing "callback"

**Comment by rrahir** [comment]
this PR should target saas-17.4, the update commit is probably not relevant  anymore :)

**Review by rrahir — CHANGES_REQUESTED** [review]

File: addons/spreadsheet/static/src/chart/odoo_chart/odoo_line_chart.js [hunk 2]
```diff
@@ -126,5 +127,21 @@ function getLineConfiguration(chart, labels, locale) {
     if (chart.stacked) {
         config.options.scales.y.stacked = true;
     }
+
+    const formatCallback = (value) => {
```
Comment (LucasLefevre) [inline addons/spreadsheet/static/src/chart/odoo_chart/odoo_line_chart.js:131]: once https://github.com/odoo/o-spreadsheet/pull/4728/files is merged, we could export the new helper from the lib and use it here

**Review by LucasLefevre — COMMENTED** [review]
small tests ? :roll_eyes:

**Comment by rrahir** [comment]
@robodoo r+

**Pull request closed without merging**